bootstrap.cmd

```diff
@@ -3,6 +3,7 @@
 where /q dotnet.exe || echo Can't find dotnet.exe! Please add to PATH && goto :EOF
 where /q git.exe || echo Can't find git.exe! Please add to PATH && goto :EOF
 set root=%~dp0
+pushd %root%
```

Comment:
If you have to change directories to run some other script, that almost always indicates a bug in the other script.  I'd like to understand more about this requirement.